REST Bindings/openapi.yaml

```diff
@@ -33,6 +29,9 @@ tags:
   - name: 'Subscriptions'
     description: |
       EPCIS 2.0 supports query subscriptions with using Webhooks or Websockets.
+  - name: 'Discovery'
+    description: |
+      Endpoints to obtain information about the endpoint, such as EPCIS and CBV versions or custom vocabularies.
```

Comment:
> @domguinard, How about adding support of dynamic EPCIS query parameter in openapi.yaml?
> 
> Here is how we can achieve this: https://stackoverflow.com/questions/49582559/how-to-document-dynamic-query-parameter-names-in-openapi-swagger
> 
> FYI: @jmcanterafonseca-iota, @mgh128

Bhavesh will prepare a prototype to illustrate this - to review on 23rd Nov call

Reply:
Hi @domguinard , @mgh128,

As discussed in our last call, I have prepared a screencast to demonstrate support of query parameters in OAS 3.0. 
I thought offline review for this will help to think and share thoughts more freely and will save time during calls as well. 

screencast link: https://drive.google.com/file/d/1I7PrI9J6_Bilp-AhZn4vIFHGgNTiIG3-/view?usp=sharing

Change to be made in openapi.yaml for endpoints supporting dynamic query parameters:

```
parameters:
        - in: query
          name: params
          schema:
            type: object
            additionalProperties:
              type: string
              additionalProperties: true
          style: form
          explode: true
```

Covered following EPCIS query in demo (Please note that as per specification ORing is possible only for multiple values of same field):

```
(perPage = 25) AND
(eventType = "ObjectEvent") AND 
(bizStep = "shipping" OR bizStep = "receiving") AND
(recordTime >= 2021-11-17T16:30:00.000Z) AND
(recordTime <= 2021-11-17T17:30:00.000Z)
```

Representation on swagger or OAS 3.0 editor:

```
{
  "perPage": 25,
  "eventType": "ObjectEvent",
  "EQ_bizStep" : "shipping|receiving",
  "GE_recordTime" : "2021-11-17T16:30:00.000Z",
  "LE_recordTime" : "2021-11-17T17:30:00.000Z"
}
```

Query parameters in URL:

```
curl -X 'GET' \
  'https://editor.swagger.io/events?perPage=25&
  eventType=ObjectEvent&
  EQ_bizStep=shipping%7Creceiving&
  GE_recordTime=2021-11-17T16%3A30%3A00.000Z&
  LE_recordTime=2021-11-17T17%3A30%3A00.000Z' \
  -H 'accept: application/json'
```

Please share your reviews.

FYI: @sboeckelmann , @jmcanterafonseca-iota

Reply:
@domguinard, @mgh128 - any thoughts?